package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
But before it's fixed contributors will see unexpected failures and possibly not see why, making for a worse contributor experience. It's not a huge deal but I just don't see why we would opt into that on purpose when there's a really easy way to avoid it.